src/snowflake/cli/app/api_impl/plugin/plugin_config_provider_impl.py

```diff
@@ -8,14 +8,24 @@
     get_config_value,
     get_plugins_config,
 )
+from snowflake.cli.api.exceptions import InvalidPluginConfiguration
 from snowflake.cli.api.plugins.plugin_config import PluginConfig
 
 
+def _assert_config_enabled_value_is_bool(value, plugin_name) -> None:
```

Comment:
Maybe lets name it `_assert_value_is_bool` as we can pass any value here

Reply:
Done